wpengine/atlas-content-modeler
# Fix memory leak when opening a field/model when its options dropdown has been activated

Fixes “Can't perform a React state update on an unmounted component” when opening an options dropdown and then opening the field/model.

This was due to attempting to set state in a setTimeout handler after the component had unmounted, and predates #90.

### To test
1. Click a field or model options to show the options dropdown.
2. Click the same field or model row to open it (without closing the dropdown seems to trigger this consistently, but also sometimes when closing the dropdown first).

You should no longer see any console warnings about an unmounted component.
